blueprints/gcve/pc-minimal/README.md

```diff
@@ -0,0 +1,120 @@
+Optionally this blueprint can deploy the VMWare Engine Network peerings to existing VPCs the user needs to create upfront.
+
+Multiple deployments of this blueprint allow the user to achieve more complex design solutions as for example GCVE private clouds deployed on different projects or connected to indipendent VMWare Engine Networks.
+
+This blueprint is used as part of the [FAST GCVE stage](../../../fast/stages/3-gcve/) but it can also be used independently if desired.
+
+<p align="center">
+  <img src="diagram.png" alt="GCVE single region private cloud">
+</p>
+
+The blueprint manages:
+- project creation
+- project-level organization policy definitions
+- billing setup (billing account attachment)
+- API/Services enablement
+- IAM role assignment for groups
+- VMware Engine private clouds creation
+- [VMware Engine Network](https://cloud.google.com/vmware-engine/docs/networking/vmware-engine-network#standard_networks) creation
+- VPC attachment (Optional)
+
+### User groups
+
+Based on our GCP best practices, a GCVE private cloud relies on user groups to assign roles to human identities. These are the specific groups bound to the main GCVE [predefined roles](https://cloud.google.com/vmware-engine/docs/iam#vmware-engine-roles):
+- *VMware Engine Administrators*. They have full access to the VMWare Engine Service.
+- *VMware Engine Viewers*. They have read-only access to the VMware Engine Service.
+
+
+### Network
+
+This blueprints expects the user to provision a VPC upfront, either from one of the FAST networking stages (e.g. [Networking with separated single environment](../../../fast/stages/2-networking-d-separate-envs)) or from an external source.
+The blueprint can optionally configure the [VMware Engine Network peering](https://cloud.google.com/vmware-engine/docs/networking/peer-vpc-network) on the peer VPC by granting the following permissions on the project that hosts the VPC:
+- vmwareengine.networkPeerings.create
+- vmwareengine.networkPeerings.get
+- vmwareengine.networkPeerings.list
+- vmwareengine.operations.get
+
+## Basic usage
+
+The following example shows how to deploy a CGVE private cloud and connect it to a VPC
+
+```hcl
+module "gcve-pc" {
+  source             = "./fabric/blueprints/gcve/pc-minimal"
+  billing_account_id = "000000-000000-000000"
+  folder_id          = "folders/000000000000"
+  project_id         = "myprojectid"
+  groups = {
+    gcp-gcve-admins  = "group:gcp-gcve-admins@acme.com"
+    gcp-gcve-viewers = "group:gcp-gcve-viewers@acme.com"
+  }
+
+  prefix = "myprefix"
+
+  network_peerings = {
+    dev-spoke-ven = {
+      peer_network                  = "projects/spokeproject/regions/europe-west1/subnetworks/dev-default-ew1"
+      peer_project_id               = "peerprojectid"
+      configure_peer_network        = false
+      peer_to_vmware_engine_network = false
+      custom_routes = {
```

Comment:
these should be handled via variable defaults, examples need to be as concise as possible

Reply:
I remove those lines from the example since defaults are already managed